core/spam/vote_spam_policy.go

```diff
@@ -433,3 +434,23 @@ func (vsp *VoteSpamPolicy) PreBlockAccept(tx abci.Tx) (bool, error) {
 
 	return true, nil
 }
+
+func (vsp *VoteSpamPolicy) GetStats(partyID string) Statistic {
+	vsp.lock.RLock()
+	defer vsp.lock.RUnlock()
+
+	stats := Statistic{
+		Limit: banFactor.String(),
+	}
+
+	bStats, ok := vsp.partyBlockRejects[partyID]
```

Comment:
how is this kind of thing ever going to be useful? this starting to be meaningful once the block has been proposed and is reset at the end of the block so what's the point of it?

Reply:
This is a misunderstanding on my part due to the ban calculation using the block stats and Gordon's comment on the original ticket regarding helping users avoiding bans. I will update in accordance to the Slack conversation.

https://vegaprotocol.slack.com/archives/CAHA5EX0F/p1673017527936189